vendor/github.com/docker/swarmkit/manager/allocator/network.go

```diff
@@ -67,8 +67,26 @@ type networkContext struct {
 	somethingWasDeallocated bool
 }
 
+// NetworkConfig is used to store network related cluster config in the Manager.
+type NetworkConfig struct {
```

Comment:
Initially Anshul suggested to keep it in manager/controlapi/network.go.  I had an issue with "cyclic import" . and discussed with him to make sure I dont break if there is any existing boundaries. We decided to implement in allocator code.  But again we had package where we got into "cyclic import" issue.  Hence I decided to add one more place to avoid this issue. I will reach out to Anshul again for his comments and update accordingly. Is that ok ?